edc-extensions/hashicorp-vault/src/main/java/org/eclipse/dataspaceconnector/core/security/hashicorp/CreateHashicorpVaultEntryRequestPayload.java

```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2022 Mercedes-Benz Tech Innovation GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Mercedes-Benz Tech Innovation GmbH - Initial API and Implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.core.security.hashicorp;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import java.util.Map;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Builder
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+@JsonIgnoreProperties(ignoreUnknown = true)
+class CreateHashicorpVaultEntryRequestPayload {
```

Comment:
I would recommend to rename the class to not have a verb in it. This should be the `HashicorpVaultEntryRequest` class.

```suggestion
class HashicorpVaultEntryRequest {
```

Reply:
The hashicorp vault API expects different payloads for creating/updating keys, also it's not responding according to that structure, so I guess it's better to express explicitly what this class is used for. Furthermore IMHO `Request` indicates the object contains all details to construct a HTTP request object - including headers, uri, method etc. - which is not the case so I'd prefer keeping that within name. 


<sub>Denis Neuling <denis.neuling@mercedes-benz.com>, Mercedes-Benz Tech Innovation GmbH, [legal info/Impressum](https://github.com/mercedes-benz/daimler-foss/blob/master/LEGAL_IMPRINT.md) </sub>

Reply:
Ahhhh got it! Then I'm totally with you ;-) Thanks for explaining this!